grpc/src/main/proto/model/workflow.proto

```diff
@@ -26,12 +26,9 @@ message Workflow {
     repeated Task tasks = 6;
     map<string, google.protobuf.Value> input = 8;
     map<string, google.protobuf.Value> output = 9;
-    string workflow_type = 10;
```

Comment:
Consider making a note here to prevent the use of these ids.

Reply:
Done.